ovf deploy crashes on apply after passing plan without issues

### Terraform Version
Terraform v0.15.0 (similar issues on 14.4

### vSphere Provider Version
v1.26.0 and v1.25.0 
connecting to vshere vSphere version 6.7.0.44000

### Affected Resource(s)
ovf_deploy
<!--- Please list the resources as a list, for example:
- `vsphere_virtual_machine`
- `vsphere_distributed_port_group`

If this issue appears to affect multiple resources, it may be an issue with
Terraform's core, so please mention this.
--->
```
### Terraform Configuration Files
terraform {
  required_providers {
    vsphere = {
      source = "hashicorp/vsphere"
      version = "1.26.0"
    }
  }
}

provider "vsphere" {
  user           = var.vsphere_user
  password       = var.vsphere_password
  vsphere_server = var.vsphere_server

  
  allow_unverified_ssl = true
}
#### RETRIEVE DATA INFORMATION ON VCENTER ####

data "vsphere_datacenter" "dc" {
  name =  var.datacenter
}
# Retrieve datastore information on vsphere (numbers start with 0)
data "vsphere_datastore" "datastore0" {
  name          = var.datastore.0
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore1" {
  name          = var.datastore.1
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore2" {
  name          = var.datastore.2
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore3" {
  name          = var.datastore.3
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore4" {
  name          = var.datastore.4
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore5" {
  name          = var.datastore.5
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore6" {
  name          = var.datastore.6
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_datastore" "datastore7" {
  name          = var.datastore.7
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_compute_cluster" "cluster" {
  name          = var.cluster
  datacenter_id = data.vsphere_datacenter.dc.id
}

data "vsphere_network" "network" {
  name = var.network1
  datacenter_id = data.vsphere_datacenter.dc.id
}
data "vsphere_host" "host" {
  name = "172.26.239.232"
  datacenter_id = data.vsphere_datacenter.dc.id
  }

resource "vsphere_virtual_machine" "gw1" {
  name             = var.names[0]
  folder         = var.folder
  resource_pool_id = data.vsphere_compute_cluster.cluster.resource_pool_id
  datastore_id = data.vsphere_datastore.datastore6.id
  host_system_id = data.vsphere_host.host.id
  datacenter_id = data.vsphere_datacenter.dc.id
  #Below to make sure it will not wait for IP address to get assigned
  wait_for_guest_net_timeout = 0
  wait_for_guest_ip_timeout  = 0
  num_cpus = 4
  memory = 4096
  
  ovf_deploy {
    allow_unverified_ssl_cert = true

    local_ovf_path = "/home/ps/tf/ovf/Check_Point_SG_R80.40.ovf"
    disk_provisioning = "thin"
    ovf_network_map = {
      "VM Network" = data.vsphere_network.network.id
    }
  }
  vapp {
    properties = {
    "guestinfo.hostname" = "bob_server",
    "guestinfo.ipaddr_v4" = "10.2.1.50",
    "guestinfo.masklen_v4" = "24",
    "guestinfo.default_gw_v4" = "10.2.1.2",
    "guestinfo.eth1_ipaddr_v4" = "10.2.2.51",
    "guestinfo.eth1_masklen_v4" = "24",
    "guestinfo.primary" = "8.8.8.8",
    "guestinfo.secondary" = "8.8.4.4",
    "guestinfo.ftw_sic_key" = "vpn123",
    "guestinfo.download_info" = "true",
    "guestinfo.upload_info" = "true",
    "guestinfo.gateway_cluster_member" = "false",
    "guestinfo.configure" = "true",
  }
}
}
```
<!--- Information about code formatting: https://help.github.com/articles/basic-writing-and-formatting-syntax/#quoting-code --->

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### Debug Output

<!---
Please provide a link to a GitHub Gist containing the complete debug output. Please do NOT paste the debug output in the issue; just paste a link to the Gist.
--->

### Panic Output

<!--- If Terraform produced a panic, please provide a link to a GitHub Gist containing
the output of the `crash.log`.--->
root@ubuntu:/home/ps/tf/ottawa/garytest# terraform apply --auto-approve
vsphere_virtual_machine.gw1: Creating...

Stack trace from the terraform-provider-vsphere_v1.26.0_x4 plugin:

panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x48 pc=0x101db80]

goroutine 42 [running]:
github.com/hashicorp/terraform-provider-vsphere/vsphere/internal/helper/ovfdeploy.(*OvfHelper).GetImportSpec(0xc00013ea50, 0xc000676790, 0xc00013ea50, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vsphere/internal/helper/ovfdeploy/ovf_helper.go:541 +0x480
github.com/hashicorp/terraform-provider-vsphere/vsphere.resourceVsphereMachineDeployOvfAndOva(0xc0006509a0, 0x130bc20, 0xc00061f6a0, 0x12a9640, 0xc0003bc660, 0x1)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vsphere/resource_vsphere_virtual_machine.go:1328 +0x14a
github.com/hashicorp/terraform-provider-vsphere/vsphere.resourceVSphereVirtualMachineCreate(0xc0006509a0, 0x130bc20, 0xc00061f6a0, 0x2, 0x216efe0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vsphere/resource_vsphere_virtual_machine.go:324 +0xa2a
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Resource).Apply(0xc000596300, 0xc000444af0, 0xc00061e680, 0x130bc20, 0xc00061f6a0, 0x133c001, 0xc0003ba438, 0xc000253b90)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/schema/resource.go:305 +0x375
github.com/hashicorp/terraform-plugin-sdk/helper/schema.(*Provider).Apply(0xc000597780, 0xc0003a38e8, 0xc000444af0, 0xc00061e680, 0xc0003ba090, 0xc0005d2e38, 0x133de20)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform-plugin-sdk/helper/schema/provider.go:294 +0x99
github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin.(*GRPCProviderServer).ApplyResourceChange(0xc00000ebf0, 0x184ef70, 0xc00012efc0, 0xc000233500, 0xc00000ebf0, 0xc00012efc0, 0xc00014ca50)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform-plugin-sdk/internal/helper/plugin/grpc_provider.go:885 +0x8a5
github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5._Provider_ApplyResourceChange_Handler(0x1586f00, 0xc00000ebf0, 0x184ef70, 0xc00012efc0, 0xc0002334a0, 0x0, 0x184ef70, 0xc00012efc0, 0xc000179900, 0x1114)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/github.com/hashicorp/terraform-plugin-sdk/internal/tfplugin5/tfplugin5.pb.go:3189 +0x214
google.golang.org/grpc.(*Server).processUnaryRPC(0xc0000b2000, 0x1857a98, 0xc000956a80, 0xc0000fc100, 0xc0005d49c0, 0x21300a0, 0x0, 0x0, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/google.golang.org/grpc/server.go:995 +0x482
google.golang.org/grpc.(*Server).handleStream(0xc0000b2000, 0x1857a98, 0xc000956a80, 0xc0000fc100, 0x0)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/google.golang.org/grpc/server.go:1275 +0xd2c
google.golang.org/grpc.(*Server).serveStreams.func1.1(0xc00034c1d0, 0xc0000b2000, 0x1857a98, 0xc000956a80, 0xc0000fc100)
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/google.golang.org/grpc/server.go:710 +0xab
created by google.golang.org/grpc.(*Server).serveStreams.func1
        /opt/teamcity-agent/work/5d79fe75d4460a2f/src/github.com/hashicorp/terraform-provider-vsphere/vendor/google.golang.org/grpc/server.go:708 +0xa5

Error: The terraform-provider-vsphere_v1.26.0_x4 plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
╷
│ Error: Plugin did not respond
│
│ The plugin encountered an error, and failed to respond to the
│ plugin.(*GRPCProvider).ApplyResourceChange call. The plugin logs may contain more details.

### Expected Behavior

I have a new vm machine built on vsphere server

### Actual Behavior

<!--- What actually happened? --->

### Steps to Reproduce
terraform apply results in the message above
<!--- Please list the steps required to reproduce the issue, for example:
1. `terraform apply`--->

### Important Factoids

<!--- Are there anything atypical about your infrastructure that we should know? --->

### References

<!---
Information about referencing Github Issues: https://help.github.com/articles/basic-writing-and-formatting-syntax/#referencing-issues-and-pull-requests

Are there any other GitHub issues (open or closed) or pull requests that should be linked here? Vendor documentation? For example:
--->

* #0000

<!--- Please keep this note for the community --->

### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

<!--- Thank you for keeping this note for the community --->

--- a/vsphere/internal/helper/ovfdeploy/ovf_helper.go
+++ b/vsphere/internal/helper/ovfdeploy/ovf_helper.go
@@ -538,8 +538,11 @@ func (o *OvfHelper) GetImportSpec(client *govmomi.Client) (*types.OvfCreateImpor
 
 	is, err := ovfManager.CreateImportSpec(context.Background(), ovfDescriptor,
 		o.ResourcePool.Reference(), o.Datastore.Reference(), importSpecParam)
+	if err != nil {
+		return nil, fmt.Errorf("while getting ovf import spec: %s", err)
+	}
 	if len(is.Error) > 0 {
-		out := "while getting ovf import spec: \n"
+		out := "while creating import spec: \n"
 		for _, e := range is.Error {
 			out = fmt.Sprintf("%s\n- %s", out, e.LocalizedMessage)
 		}

